File: src/test/java/org/jdc/template/ux/individual/IndividualPresenterTestComponent.java
package org.jdc.template.ux.individual;


import org.jdc.template.inject.CommonTestModule;

import javax.inject.Singleton;

import dagger.Component;

@Singleton
@Component(modules = {IndividualPresenterTestModule.class, CommonTestModule.class})
public interface IndividualPresenterTestComponent {
    void inject(IndividualPresenterTest target);
}

File: src/test/kotlin/org/jdc/template/ux/individual/IndividualPresenterTestComponent.kt
package org.jdc.template.ux.individual

import dagger.Component
import org.jdc.template.inject.CommonTestModule
import javax.inject.Singleton

@Singleton
@Component(modules = arrayOf(IndividualPresenterTestModule::class,
        CommonTestModule::class))
interface IndividualPresenterTestComponent {
    fun inject(target: IndividualPresenterTest)
}